Description: apparmor entries for vfio hotplug

vfio devices are generated on the fly, but the generic base is missing.

Note: to be upstreamed with all the other apparmor changes as soon as we get
to upstream the base work on these.

Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1680384
Forwarded: no
Last-Update: 2017-04-18
--- a/examples/apparmor/libvirt-qemu.in
+++ b/examples/apparmor/libvirt-qemu.in
@@ -211,3 +215,6 @@
 
   # allow connect with openGraphicsFD to work
   unix (send, receive) type=stream addr=none peer=(label=/usr/sbin/libvirtd),
+
+  # for vfio (LP: 1678322)
+  /dev/vfio/vfio rw,
